package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "s47_most_searched_website_on-google",
+  "version": "1.0.0",
+  "description": "Idea title : Most googled website in 2023",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
```

Comment:
It would be a good idea to add more information in the package.json file, such as the "author" and "description" fields. This can help others understand more about the project. [medium]